activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ConnectorValidationControllerIT.java

```diff
@@ -252,14 +252,50 @@ public void should_returnStatusNoContent_when_validatingConnectorWithCustomTypes
         final Model connectorModel = modelRepository.createModel(connectorModel("connector-name"));
 
         given()
-            .multiPart("file",
-                "connector-with-custom-type.json",
-                resourceAsByteArray("connector/connector-with-custom-type.json"),
-                "text/plain")
-            .post("/v1/models/{modelId}/validate",
-                connectorModel.getId())
-            .then()
-            .expect(status().isNoContent())
-            .body(isEmptyString());
+                .multiPart("file",
+                        "connector-with-custom-type.json",
+                        resourceAsByteArray("connector/connector-with-custom-type.json"),
+                        "text/plain")
+                .post("/v1/models/{modelId}/validate",
+                        connectorModel.getId())
+                .then()
+                .expect(status().isNoContent())
+                .body(isEmptyString());
+    }
+
+    @Test
+    public void shouldReturnStatusNoContentEhenValidatingConnectorWithErrors() throws IOException {
+        final Model connectorModel = modelRepository.createModel(connectorModel("connector-name"));
+
+        given()
+                .multiPart("file",
+                        "connector-with-events.json",
+                        resourceAsByteArray("connector/connector-with-events.json"),
+                        "text/plain")
+                .post("/v1/models/{modelId}/validate",
+                        connectorModel.getId())
+                .then()
+                .expect(status().isNoContent())
+                .body(isEmptyString());
+    }
+
+    @Test
+    public void shouldThrowSemanticValidationExceptionWhenValidatingInvalidConnectorErrorInvalidProperty() throws IOException {
```

Comment:
same as above

Reply:
Yep I thought the same and was weird to me that he changed the naming in the last commit, but when I talked to @rbarroso, he told me that there is a rule that looks for the naming convention in methods that is applied also in test methods for writing them in camel case.

Reply:
@erdemedeiros did not we agree on this format at some point? should_returnStatusNoContent_when_validatingConnectorWithErrors 
in enterprise we are still using it

Reply:
Exactly @erdemedeiros. Because of that warning I have renamed the methods. 
I will update them to should_returnStatusNoContent_when_validatingConnectorWithError.
Thanks all.